src/react/components/pages/editorPage/editorPage.test.tsx

```diff
@@ -138,12 +142,12 @@ describe("Editor Page Component", () => {
             name: testProject.name,
         };
 
-        const expectedAssetMetadtata: IAssetMetadata = getMockAssetMetadata(testAssets);
-
         await MockFactory.flushUi();
 
+        const expectedAssetMetadtata: IAssetMetadata = getMockAssetMetadata(projectAssets);
```

Comment:
what prompted the order of operation change?

Reply:
Everything was working fine, just my OCD since all the expectations happen after the UI has been updated.